File: Daily Challanges/2026-04-April/2026-04-19.js
function getUniqueClimbs(steps) {
  let table = [1, 2];
  for (let i = 2; i < steps; i++) table.push(table.at(-2) + table.at(-1));
  return table.at(-1);
}

console.log(getUniqueClimbs(4));
console.log(getUniqueClimbs(5));
console.log(getUniqueClimbs(10));

File: Daily Challanges/2026-04-April/2026-04-19.py
def get_unique_climbs(steps):
    table = [1,2]
    for step in range(2, steps):
        table.append(table[-1] + table[-2])
    return table[-1]

print(get_unique_climbs(4))
print(get_unique_climbs(5))
print(get_unique_climbs(10))